### PR TITLE
tipb: add index req and resp in analyze columns

### PR DESCRIPTION
**Pull request by lzmhhh123**

### CODE REVIEW

File: proto/analyze.proto [hunk 1]
```diff
@@ -25,7 +26,7 @@ message AnalyzeReq {
     optional uint64 start_ts_fallback = 2;
     optional uint64 flags = 3 [(gogoproto.nullable) = false];
     optional int64 time_zone_offset = 4 [(gogoproto.nullable) = false];
-    optional AnalyzeIndexReq idx_req = 5;
+    repeated AnalyzeIndexReq idx_req = 5;
```
Comment (coocood) [inline proto/analyze.proto:29]: This may break compatibility?

Comment (lzmhhh123) [reply proto/analyze.proto:29]: Yes.

File: proto/analyze.proto [hunk 2]
```diff
@@ -66,6 +69,12 @@ message AnalyzeColumnsReq {
     repeated int64 primary_column_ids = 7;
 }
 
+message AnalyzeResp {
```
Comment (coocood) [inline proto/analyze.proto:72]: How about `AnalyzeMixedResp`?

File: proto/analyze.proto [hunk 3]
```diff
@@ -43,6 +44,8 @@ message AnalyzeIndexReq {
     optional int64 sample_size = 5 [(gogoproto.nullable) = false];
 
     optional int64 sketch_size = 6 [(gogoproto.nullable) = false];
+
+    optional IndexInfo index_info = 7;
```
Comment (coocood) [inline proto/analyze.proto:48]: Can we remove it?

Comment (lzmhhh123) [reply proto/analyze.proto:48]: Yes, we could.

**Comment by coocood** [comment]
LGTM

**Review by tangenta — COMMENTED** [review]
LGTM

**Review by coocood — APPROVED** [review]

**Pull request merged**